app/containers/MessageBox/buttons/OpenFullScreenButton.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import BaseButton from './BaseButton';
+
+const OpenFullScreenButton = React.memo(({ theme, onPress }) => (
```

Comment:
Is memo actually needed here?

Reply:
This memo is to improve the performance, these props don't often change then we can win with this memo. I created this button based in all other buttons.

Reply:
I understand, but is there actually any measured performance gains, or is it just in theory? BaseButton also uses `React.memo`, so do we still need it at this level?
Is this component actually required since it is only used in one place and doesn't wrap anything complicated?

Reply:
Got it. make sense because it was just a theory but this `React.memo` is not necessary in this button. thanks :)